packages/verticaltabs/src/react/item.tsx

```diff
@@ -242,6 +242,11 @@ const Tier2Header = React.forwardRef<any, ItemHeaderProps>((props, ref) => {
       <span {...wrapperProps} ref={ref as RefFor<'span'>} />
     )
 
+  if (props.collapsed || hideLabels) {
+    rest['aria-hidden'] = 'true'
+    rest.tabIndex = -1
+  }
+
```

Comment:
I'd prefer it if you create a new object here instead of mutation the props and passed that object as a spread after rest.